src/index.tsx

```diff
@@ -4,7 +4,13 @@ import rough from "roughjs/bin/wrappers/rough";
 
 import { moveOneLeft, moveAllLeft, moveOneRight, moveAllRight } from "./zindex";
 import { randomSeed } from "./random";
-import { newElement, resizeTest, isTextElement, textWysiwyg } from "./element";
+import {
+  newElement,
+  duplicateElement,
```

Comment:
Can you also use duplicating elements for the paste logic?

Reply:
Done. Turned out on paste we weren't regenerating ids, either, so good call.